### PR TITLE
docs: Update documentation screenshots

### PR DESCRIPTION
**Pull request by ashuntu**
Updates the screenshots for the Ubuntu Pro GUI app in the docs to the most recent designs.

---

UDENG-5685

### CODE REVIEW

**Comment by ashuntu** [comment]
I don't believe the CI failure is a result of these changes.

**Review by edibotopic — CHANGES_REQUESTED** [review]
All looks good but I have one query on the status page after configuration through the registry.

Also, I just re-ran the docs CI and it fixed the CI error -- sometimes it needs some massaging :smiling_face_with_tear:

File: docs/howto/assets/status-complete-registry.png
[the remark this answers is not in the document]
Comment (CarlosNihelton) [reply docs/howto/assets/status-complete-registry.png:1]: Well noticed! Here's how the app looks when the registry is fully set (running from the tip of the main branch):

image

[the remark this answers is not in the document]
Comment (ashuntu) [reply docs/howto/assets/status-complete-registry.png:1]: Good point. I'll update that screenshot accordingly

**Review by edibotopic — APPROVED** [review]
All good now. Thanks for this, Ashton!

**Pull request merged**